packages/jsii-pacmak/lib/targets/go/runtime/index.ts

```diff
@@ -60,22 +65,37 @@ export class ClassConstructor {
 
   public emit(code: CodeMaker) {
     emitInitialization(code);
-    code.open(`jsii.NoOpRequest(jsii.NoOpApiRequest {`);
-    code.line(`Class: "${this.parent.parent.name}",`);
-    code.line(`Method: "Constructor",`);
-    code.line(
-      `Args: ${paramsString(
-        this.parent.parent.type.initializer!.parameters.map((p) =>
-          p.type.toString(),
-        ),
-      )},`,
-    );
-    code.close(`})`);
+    code.line(`returnval := ${this.parent.parent.name}{}`);
```

Comment:
So we need to actually use a name here that will never conflict with any parameter of the constructor... Either some name that just cannot be a parameter name, or slugify appropriately if a parameter matches...

---

Alternatively (and perhaps this is a batter option?), make `JSII_CREATE_FUNC` return the `returnval`, so you can (⚠️ heavy schematization ahead):
```go
return JSII_CREATE_FUNC(
    "parent.parent.fqn",
    []args{},
    []interfaces{},
    []overrides{},
    &parent.parent.name{}
)
```